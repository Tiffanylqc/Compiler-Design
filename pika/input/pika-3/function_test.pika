func fin< int a, int b >-> int{
	return pax(5*a,b).
	var c:=4.
	return b.
}
func pax < int a, int b >-> int{
	if(a<b){
		return a.
	}
	return b.
}
func foo <>->void{
	return.
}
func foo1 <>->void{
	return.
}