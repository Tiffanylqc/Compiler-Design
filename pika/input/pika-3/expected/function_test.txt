Couldn't read file ".\output\function_test.asm"
